src/rdb.c

```diff
@@ -2022,6 +2022,7 @@ void startLoading(size_t size, int rdbflags) {
     server.loading_start_time = time(NULL);
     server.loading_loaded_bytes = 0;
     server.loading_total_bytes = size;
+    server.rdb_expired_keys_last_load = 0;
```

Comment:
maybe we rather let this keep incrementing forever?
i.e. resetting it may be nicer as a single metric to look at with redis-cli.
not resetting it may be nicer for plotting this metric in some monitoring software (e.g. Grafana)

if we don't reset it, we should obviously change the field name.

Reply:
now that we don't rely on this metric, i really think it should be zeroed in resetServerStats

Reply:
Still need to remove that reset from startLoading